android-sample-java/app/src/main/java/io/logto/demo4j/viewmodel/LogtoViewModel.java

```diff
@@ -0,0 +1,97 @@
+package io.logto.demo4j.viewmodel;
+
+import android.app.Activity;
+import android.app.Application;
+import androidx.annotation.NonNull;
+import androidx.lifecycle.AndroidViewModel;
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import io.logto.sdk.android.LogtoClient;
+import io.logto.sdk.android.exception.LogtoException;
+import io.logto.sdk.android.type.AccessToken;
+import io.logto.sdk.android.type.LogtoConfig;
+import io.logto.sdk.core.type.IdTokenClaims;
+
+public class LogtoViewModel extends AndroidViewModel {
+
+    public LogtoViewModel(@NonNull Application application) {
+        super(application);
+    }
+
+    private LogtoConfig logtoConfig = new LogtoConfig(
+            "https://logto.dev",
+            "z4skkM1Z8LLVSl1JCmVZO",
```

Comment:
Do we need a database seed issue to ensure the clientId is valid on user's self-hosted Logto core?

Reply:
it's more like a doc issue? let's figure out it later

Reply:
LOG-1554